jupyterhub_moss/spawner.py

```diff
@@ -233,29 +234,30 @@ async def _get_slurm_info_resources(self):
                 errmsg = "Missing required resource counter in Slurm partition: {}"
                 raise KeyError(errmsg.format(partition))
 
-        return (resources_display, resources_info)
+        partitions_info = {}
+        for partition, description in self.partitions.items():
+            partitions_info[partition] = resources_info[partition]
+            # use data from Slurm as base and overwrite with manual configuration settings
+            partitions_info[partition].update(description)
```

Comment:
Overrides are moved here, so this method can be called consistently from multiple places.

Reply:
`config_partition_info` ?